lib/palettes_base.cpp

```diff
@@ -443,18 +443,16 @@ void PaletteRegistry::Load(const string& palette_filename)
       {
          pfile >> palname >> channeltype;
          idx = GetIndexByName(palname);
-         if (idx == -1)
+         if (idx != -1)
          {
-            AddPalette(palname);
-            idx = GetIndexByName(palname);
-            cout << "Reading palette: (" << idx+1 << ") " << palname << endl;
-         }
-         else
-         {
-            cout << "Error reading palette: " << palname
-                 << ". Palette with same name already exists." << endl;
-            break;
+            cout << "Warning: palette name <" << palname
+                 << "> already exists. Overriding..." << endl;
+            palettes.erase(palettes.begin() + idx);
```

Comment:
As it is a good practice of premature optimization in MFEM, it would be more optimal if you do `palettes[idx] = as_unique<Palette>(palname)` 😄 . But for unification of the approach it is better like this. Would be better to wrap this call in a method, but also not super important.